Packages/vcs/Lib/VTKAnimate.py

```diff
@@ -0,0 +1,81 @@
+# VTK Backend Animation Module
+## Author:  Charles Doutriaux
+import animate_helper
+import vcs
+import time
+import random
+import hashlib
+import os
+import glob
+
+class VTKAnimationCreate(animate_helper.StoppableThread):
+  def __init__(self, controller):
+    animate_helper.StoppableThread.__init__(self)
+    self.controller = controller
+
+  def run(self):
+    self.controller.animation_created = True
+    self.controller._unique_prefix=hashlib.sha1(time.asctime()+str(random.randint(0,10000))).hexdigest()
+
+  def describe(self):
+    for info in self.controller.animate_info:
+      disp = info[0]
+      print "BACKEND:",disp.backend
+      print "TYPE:",disp.g_type
+      print "Name:",disp.g_name
+      if info[1][0] is not None:
+        print "Array:",info[1][0].shape
+      else:
+        print "No Array"
+
+class VTKAnimationPlayback(animate_helper.AnimationPlayback):
+  def __init__(self, controller):
+    animate_helper.AnimationPlayback.__init__(self,controller)
+
+class VTKAnimate(animate_helper.AnimationController):
+    def __init__(self,vcs_self):
+        animate_helper.AnimationController.__init__(self,vcs_self)
+        self.AnimationCreate = VTKAnimationCreate
+        self.AnimationPlayback = VTKAnimationPlayback
+        self.cleared = False
+        import atexit
+        atexit.register(self.close)
+    def draw_frame(self):
+      png_name=os.path.join(os.environ["HOME"],".uvcdat",self._unique_prefix,"anim_%i.png" % self.frame_num)
+      if os.path.exists(png_name) and len(self.animation_files)==self.number_of_frames():
+        ## Ok we have the pngs and we need to zoom, need to use png
```

Comment:
Can we get an option to require that we don't use PNGs? This will help a lot with scrubbing the animation from the VCS2D interactive code.